staging/src/k8s.io/apiserver/pkg/storage/etcd3/store_test.go

```diff
@@ -927,7 +925,7 @@ func TestList(t *testing.T) {
 				Field:    fields.OneTermEqualSelector("metadata.name", "bar"),
 				Label:    labels.Everything(),
 				Limit:    2,
-				Continue: encodeContinueOrDie("meta.k8s.io/v1", int64(continueRV), "z-level/3"),
+				Continue: storagetesting.EncodeContinueOrDie("z-level/3", int64(continueRV)),
```

Comment:
This test should not be concerned with what is an appropriate API version or not, and should use the public method to encode.